### ISSUE TITLE
Falling into the void in the end while under the island and preventFallingThroughBedrock is enabled teleports you back up

### ISSUE THREAD

**Issue by Ameliorate** [civmc/Bumhug#15]

**Comment by soerxpso** [comment]
> Soer likes this so much that if anyone fixes it it needs to have a config option to unfix it.

Correct. It sounds like something really fun for anyone who knows about it and uses it in combat, and affects only a small subset of combat encounters (specifically, an unimportant subset). If there is another server that really wants this fixed, I recommend changing preventFallingThroughBedrock to be a cleaner fix without side effects, and changing this one to be called "teleportIfBelowBedrock" or something.
